optuna/notebooks/optuna_rapids.ipynb

```diff
@@ -0,0 +1,687 @@
NVeI7X1mgGv2pVSqPfektU4GdARnckueEC8pZaSicRHm47hcL6R1SMJHZ3Gg/3v4QG09q5D37gwJIte2zkzPAK/qw4XILu4GuH+Bgxo7Ytb+9ePzZk21Z5zKUV7aU5qgK84MU2JwmBvfexFe7k729nPp7KmpCoPzlIeiL5ggh6h3nVW9AUe8VUDfImv+/UfKXhy+X7m1ta6ZTDeW30AI7qE4pYJPdl4pXQD+owA7aCJL6Bvv474++OGJhX21ozALRUCuEVCnhlImUFLhHnSqzWYEe5ngEGrRWsRPaGgpAKlVbUs6UlKXVDq09GHZFNEfaV83b6TX8SJEjB7YKIwOOuYJtbSLaioUeToyln/9k4Iwse/n8IzU7tict9YyCWhiRPQ5KK50mtC8mMEdh8f3NqKwkD11u87icNFZkzqGIrY8EDYiupSfTFv9931h/H19izMv74j4sL8LFbB9u7Di/242CCCktAI6BJf/fVJtrP8ia9bXadXlZzGk9mc4fXS+l1IioOje9ne9b37u0P2qjTK8QWTBW3p5uK+FWgGvC6uJd9cG1Oz8MPRIgZ86VUjlZ6tAlnkttpUh+PnRes5LSG9oBIH8oyM+kDFz+DBvsT72PhrXM1QpUloK3acwfFSgBLd7On7qmlfTZ1LLdpLc1ICvnKJaWrWQK6OrYQ2qs8pDp/P6MeUEqiIVR4o6jukfTiu6NLK7hB4UpoeRsRHBSrWF2v4to0OwKhuLRlvlgpRDchZr8AsSBTtTcnB6j9TkZwUibdv7YF/dhzD3FVH8PdbV8NWdFZtMhsB3IyiSpjMkAW4Wg/AX29GdKC3FcDlkxMD3UCDGSEB8qYCcovhykPyQkR9F/yThjPldUxfN+3wMVw/+yumwuAKhUGspVtQXsWAbpXGC3H+egwSJW4Nnr2KGUVQIhpJi33/+BD4B3rhvV/SsHZLOva/N74B0FUbzRVfC7IHfuiHFHQKN2D+lF4NLpMU7NqK6tLJBHZr6zyQll2C+785bDGP4O5pl3pklwNdmisZRLQI8Yc9m2C+qGK+rt0vk/MVXOX18mjvq6Pbq+3SqXqu7GW5Du9Z2TSA95NrmwGvUzeAjZOaAa+LK8o3F9ESKMEspQQw1taxKFT7oHq1BOLc2jKG+DslC/szy5BaaGS0ByripDdbw7SXhMY1e3tF++BYkTpzCxeXhZ1+qUV7xfze+wa3QbdWwqthZ6O6SmtoK9rLQa+cOxuP+lK7SpQHMX1BmpRmKxHs8KkirN9+Fp3igvHypCQQbdzXxwcVZSXolSBYS/MiTmob0D4cDy7able3l/N6K/d8YGlHCnBJMaHMpIFeU4cAA1Bq1NgEuO4AurwNdzwkOfD109UhxEfntiQ3Sk6jP6KvSIrC54vXW4wk7ryqE8Z1beEwhYF4uhq9FzONUAK6UuvffknhuG1kOwSH+KHICKz77yRysgotVsG0js6AXL7+BHafXHec2dw+cEXDV7lisHsio6SBAoN0rxGY33GmAl9uSmWH7h4chy7RgSiv1TKr4EsV7EqjuRzo0hwJ7HaM9MVtA+uTpaXrYouvq/Y73x283sZUcXDHXhavxb2rDqtdGrfW+/gaIaG5ubhvBZoBr4trOXXeYrTp0kegJPjUoWukDwZ1aAMe8W0f6IHyaiMKzQbVNr8EfqlQ0pst8OtIEhoHvaTi0KO1EPEjcwuK+lKZOUTencjF5WGnz165F4dzKrF2Rn93NNfobSz8MxUfbz6LgfHBmH9TcqP0p4biYMudTUp5uLF/DMrKy3GmUgeYzagzC3/YKFENlHizPOI7sU8bvHJtEny9hftDrkyc+wuL/JF2b35xBQO+X8+6HJMHy0dwXly0Aa9++BOef2gikrp3YpJgROnRCTlnFoArR1OQ69+ViK60PXc+JN0lbcb5uiE6E0tOc5XCIAa6uaWV2HUqt0FEV2r9S45oHOjSmpHawmcbjmDL0VwL2LWXgGZvA5G27t6MCgxp7SkLdg+ezMKvpypwRYwPVm87x9zSuK6uUtvv/HYci/87xegLJDPGrYJ9tbVMOu9SK1KgO6BdvVYzV2IgvrMasGuLr6t2XVzh9VIfjUlxcOdeprHOWN00gPejSc2AV+39qLZeM+BVu1IK9RZ+8yNO+CehQ6get/St58nyaO6xQhOq6B0t0IDnq6ZrKfg1Go0I8vNyOgnt+Z8PWak42DK3UDM+tXUulWivODFtY2ou5v2ajhHtgvD6DT3UTtWhehz4yiW08WjvljfGsDbFVAMOZk/mVrBj247lIy7CD31a++KytqEOJYKJB8z1dX/aeUaVa5utaC9PMOP8WwK4fppaDLr+VYwakIj7Zgp8ZVs0hcYGurx9dz8kebvOSpuJ+brawnwLheG2G4fB0cQ0scSYPaArtv4dMyAOEVHBVklozy7ZysDud48NR0G50WIOoYabK72WFNX98VAOA7tR3sBHUwU+uLgcyyrCmpQC+FWX44ftmewQ19WVuze4+kL7UB2W33c5q3Ipg12ehCY1iOBzl9oEK335OMLXVfsF5iqvlygOJwurcP9g9wZb3L2XZ34v6Dlf6PLBRNua/hd6PP8f+msGvC5cRYrc3vXuCrTt1gcvXNneJmWBR3ypO2mCm5ohUDR35SFBmzS/wiSr9aumHfbAIO/0HdlMxeFyEVdPKnHWGFHfizXaa0tuTE1im9q1l6v37He7seDXExjR3h9XD01iVXhkdm9KLr7/KxXJ7SNx39j2VjJdxJsVa+r+nZLjksqDnL6umONry6547ne78NrqQxjZKRgxbVrg4+/34Pkbk3Hd5YnIKali8mCmujpLFPeFeUug9w3AmvfUO7O5M6IrvQ7ufkhK23ck6svBLvF1F3z+Exas2o5Bk67E2P4JeHCE+oegWqD70vKdzGWP+Lk3DY0HAV2iLZBJxNiu9XzxEXN+YgB3wX2XQ2M2ysqJqd0HK3acZNq6QTozqmpMNsHu9kNZ+GtPhs2oLldf0Oo9cUP3E
```

Comment:
Not sure this cell is really necessary. We don't articulate why one might prefer this over dask-optuna, and adding a second path seems potentially confusing.